code/src/components/Album.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Icons } from "components/Icons"
+
+export const Album = (props) => {
+    return (
+        <div className='artistBox'>
```

Comment:
remember, we're working with React but you can still use semantic HTML elements such as section, article and so on. Just to think of in the future.